programs/keeper/keeper_main.cpp

```diff
@@ -154,6 +154,7 @@ __attribute__((constructor(0))) void init_je_malloc_message()
 #endif
 
 /// OpenSSL early initialization.
+/// Must be ran after EnvironmentChecks.cpp, as OpenSSL uses SSSE3.
```

Comment:
@thevar1able

The comment seems a bit confusing to me.

1. We better reference "202" and "the pre-main initialization in EnvironmentChecks" in l. 157
2. More importantly, is there evidence that OpenSSL _requires_ SSE3? (perhaps some SSE3 instructions are used in the assembly files that are included by contrib/openssl-cmake/CMakeLists.txt). Asking because the "amd_compat" build in CI ([example](https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=79097&sha=latest&name_0=PR&name_1=Build+%28amd_compat%29)) builds with `-DNO_SSE3_OR_HIGHER=1` (build log) and which _would_ break OpenSSL if it needed SSE3.